api/src/controllers/login.js

```diff
@@ -551,30 +542,39 @@ module.exports = {
     }
   },
   oidcLogin: async (req, res) => {
-    req.body = { locale: 'en' };
+    const limited = await rateLimitService.isLimited(req);
+    if (limited) {
+      return serverUtils.rateLimited(req, res);
+    }
```

Comment:
I think we should rate-limit these endpoints that are called _pre-authorization._